src/Tools/AI Test Toolkit/src/AITTestContextImpl.Codeunit.al

```diff
@@ -186,6 +195,55 @@ codeunit 149043 "AIT Test Context Impl."
         AITTestSuiteMgt.EndRunProcedureScenario(AITTestMethodLine, AITALTestSuiteMgt.GetDefaultRunProcedureOperationLbl(), TestMethodLine, ExecutionSuccess);
     end;
 
+    /// <summary>
+    /// Initializes global variables for the iteration.
+    /// </summary>
+    local procedure InitializeGlobalVariables()
+    var
+        TestInput: Codeunit "Test Input";
+        TestInputJson: Codeunit "Test Input Json";
+    begin
+        CurrentTurn := 0;
+        GlobalTestOutputJson.Initialize();
+        TestInputJson := TestInput.GetTestInput().ElementExists(TurnsTok, IsMultiTurn);
```

Comment:
Can we rename this? `TurnsInputJson` maybe? I had a hard time finding where `IsMultiTurn` was set initially.

`TestInputJson` made it sound like it contained the entirety of the test input json.